convert/src/vertex_to_bd/badgerdoc_format/plain_text_converter.py

```diff
@@ -0,0 +1,78 @@
+from typing import List
+
+from ..models import BadgerdocToken, Offset, Page, PageSize
+
+DEFAULT_PAGE_BORDER_OFFSET = 15
+
+
+def generate_chunks(text: str, n=70) -> List[str]:
+    return [text[i : i + n] for i in range(0, len(text), n)]
+
+
+class PlainTextToBadgerdocTokenConverter:
```

Comment:
May you make class name more "elegant"?

Reply:
Removed `Plain` and added `s` to `Token`
You are welcome to share any ideas.